### ISSUE TITLE
Finetuning is reducing the ROI area and cutting parts of the object

### ISSUE THREAD

**Issue by ifahim** [ultralytics/yolov5#1631]
## ❔Question
I generated a yolov5 model for person detection on my custom data. The generated bounding boxes looked okay from the output of the model on test data with the objects completely inside the BBox. Now I  got some more data and finetuned  the previous model on the new data for more number of epochs. 

What I see that the ROI  is becoming tighter and  fitting the objects better. However, in some cases, the ROI is so tight that it starts missing some parts/limbs of person. 

What possibly may be the reason for missing those limbs and how to get around this issue?

**Comment by glenn-jocher** [comment]
@ifahim improving results is always the same workflow, which I've repeated many times here:

- Use a larger model
- Use a larger dataset
- Improve labels
- Train longer
- Use hyperparameter evolution
- Use a larger image size
etc 
etc

**Comment by ifahim** [comment]
@glenn-jocher Thanks for input
